CHANGELOG.md

```diff
@@ -1,81 +1,81 @@
+- `menuButtonLabel` option to set the `aria-label` text for the button that hides or shows the navigation section on mobile
 
 For example:
 
 ```javascript
-{{ govukInput({
-  label: {
-    text: "Amount, in pounds"
-  },
-  prefix: {
-    text: "£"
-  }
+{{ govukHeader({
+    navigationLabel: "Custom navigation section aria-label",
+    menuButtonLabel: "Custom menu button aria-label"
 }) }}
 ```
 
-You shouldn't rely on prefixes or suffixes alone because people who use a screen reader won't see them. You should add any specific type of information you need to the input label or hint text as well.
-
-This was added in [pull request #1816: Add input prefix and suffix](https://github.com/alphagov/govuk-frontend/pull/1816).
+The default labels are now:
 
-#### Add HTML test fixtures
-You can use our test fixtures to check you're outputting the same HTML that GOV.UK Frontend uses.
+- **Navigation menu** for `navigationLabel`
+- **Show or hide navigation menu** for `menuButtonLabel`
 
-This was added in [pull request #1925: Generate fixtures.json files for components on build:package](https://github.com/alphagov/govuk-frontend/pull/1925)
-
-#### Add new brand colour for The Foreign, Commonwealth and Development Office (FCDO)
+This was added in pull requests:
 
-This was added in [pull request #1918: Add new brand colour for FCDO](https://github.com/alphagov/govuk-frontend/pull/1918).
+- [#1905: Set navigation and mobile menu labels of the header component with new options](https://github.com/alphagov/govuk-frontend/pull/1905)
+- [#1943: Change header menu button label](https://github.com/alphagov/govuk-frontend/pull/1943)
 
-#### Render header navigation item without a link
+Thanks to [@domoscargin](https://github.com/domoscargin) for raising this issue.
 
-You can now render navigation items using the header macro without wrapping them in a link.
+##### Add navigation items without links
 
-You can do this by setting `text` or `html` and not setting `href`.
+To add a navigation item without a link, use the `text` or `html` option to add the item but do not use the `href` option.
 
 For example:
 
 ```javascript
 {{ govukHeader({
-  navigation: [
+    navigation: [
     {
-      html: '<form method="post" action="url.com">
-              <input type="submit" class="app-logout-button-style" value="Log out" />
-            </form>'
+      html: "<form method='post' action='url.com'>
+              <input type='submit' class='app-logout-button-style' value='Log out' />
+            </form>"
     }
   ]
 }) }}
 ```
 
-This was added in [pull request # 1921: Make it possible to exclude link from header navigation item](https://github.com/alphagov/govuk-frontend/pull/1921).
+This was added in [pull request #1921: Make it possible to exclude link from header navigation item](https://github.com/alphagov/govuk-frontend/pull/1921).
 
-#### Set navigation and mobile menu labels of the header with new options
+#### Test if your HTML matches GOV.UK Frontend
```

Comment:
This is just a suggestion but I would be tempted to position this before the header features as it feels like the more important feature.